packages/components/src/FormField/FormField.tsx

```diff
@@ -233,7 +233,7 @@ export function FormField({
   );
 
   return (
-    <>
+    <div>
```

Comment:
Does this need to be a div?

Reply:
This is done so that when using something that has `display: flex` (such as `Content`) the validation message and the field are treated as one element. Without this, the validation message will get equal vertical spacing as 2 fields.